course5/homework5/vector.h

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+
+namespace vector{
+
+    class Vector2D
+    {
+    public:
+            Vector2D() = delete;
+            Vector2D(double, double);
+            Vector2D(const Vector2D&);
+        //    Vector2D(Vector2D&&);
+            Vector2D& operator=(const Vector2D&);
+        //    Vector2D&& operator=(Vector2D&& src);
+            double GetX() const;
+            double GetY() const;
+            void SetX(double);
+            void SetY(double);
+            double GetR() const;
+            double GetAngle() const;
+            Vector2D operator+(const Vector2D&);
```

Comment:
You mean that the method is const, so
Vector2D operator+(const Vector2D&) const; ?